vocabularies/Common.json

```diff
@@ -1546,6 +1546,18 @@
             "@Common.Experimental": true,
             "@Core.Description": "Sorting and filtering of amounts in multiple currencies needs special consideration",
             "@Core.LongDescription": "TODO: add link to UX documentation on https://experience.sap.com/fiori-design/"
+        },
+        "mediaUploadURL": {
```

Comment:
```suggestion
        "MediaUploadURL": {
```
Most other terms in this vocabulary are capitalized, so I'd suggest doing this here as well.

Reply:
All newer instance annotations in this vocabulary are lowerCamelCase.

Reply:
Maybe we should actually call it `mediaUploadLink` to be in sync with the `mediaReadLink` and `mediaEditLink`.

Reply:
Ok, I did not know about the naming convention for instance annotations.
Yes, mediaUploadLink might make sense then.

Reply:
Could a mediaUploadLink also point to an HTML page where users must use an HTML form with a file uploader to submit their new file? That would be clear. Or is the expectation that the file is uploaded via a REST call?